service/bookturks/quiz/QuizTools.py

```diff
@@ -22,24 +22,6 @@ def __init__(self):
         self.dbx = DropboxClient(settings.DROPBOX_CLIENT)
 
     @staticmethod
-    def get_quiz_id(username, quiz_name):
-        """
-        Concatenate to form the quiz_id
-        :param username:
-        :param quiz_name:
-        :return:
-        """
-        if not quiz_name.rstrip():
-            raise ValueError("Quiz Name cannot be blank")
-        quiz_id = "_".join([username, quiz_name.lower()])
-        if re.match("^[A-Za-z0-9_ -]*$", quiz_name):
```

Comment:
@zalokhan added the check in new commit https://github.com/zalokhan/BookTurks/pull/31/commits/82392109a9485fd65bcf4b26a0ac3ce231554b9a